### PR TITLE
tools/system/supervise: init at 1.0.0

### PR DESCRIPTION
**Pull request by catern**
###### Motivation for this change

Add supervise utility https://github.com/catern/supervise

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/system/supervise/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    mkdir -p $out/bin
+    cp supervise unlinkwait -t $out/bin
+  '';
+
+
+  meta = {
+    homepage = https://github.com/catern/supervise;
+    description = "A minimal unprivileged process supervisor making use of modern Linux features";
+    platforms = stdenv.lib.platforms.linux;
+    license = stdenv.lib.licenses.gpl3;
```
Comment (Mic92) [inline pkgs/tools/system/supervise/default.nix]: Do you want to maintain this package?

File: pkgs/development/python-modules/supervise_api/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  propagatedBuildInputs = [ supervise ];
+
+  # no tests
+  doCheck = false;
+
+  meta = {
+    description = "An API for running processes safely and securely";
+    homepage = https://github.com/catern/supervise;
+    license = with lib.licenses; [ lgpl3 ];
```
Comment (Mic92) [inline pkgs/development/python-modules/supervise_api/default.nix]: Do you want to maintain this package?

File: pkgs/tools/system/supervise/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  installPhase = ''
+    mkdir -p $out/bin
+    cp supervise unlinkwait -t $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/catern/supervise;
+    description = "A minimal unprivileged process supervisor making use of modern Linux features";
+    platforms = platforms.linux;
+    license = licenses.gpl3;
```
Comment (Mic92) [inline pkgs/tools/system/supervise/default.nix]: Do you want to maintain this package?

Comment (catern) [reply pkgs/tools/system/supervise/default.nix]: Sure, I can do that.

File: pkgs/development/python-modules/supervise_api/default.nix (hunk 2 above)
Comment (catern) [reply pkgs/development/python-modules/supervise_api/default.nix]: Sure. I will add myself to the maintainers list.

File: pkgs/development/python-modules/supervise_api/default.nix [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
```
Comment (vyp) [inline pkgs/development/python-modules/supervise_api/default.nix]: Why not just `license = lib.licenses.lgpl3;`?

**Comment by FRidh** [comment]
Thanks. Pushed to master.

**Pull request closed without merging**